pkg/cmd/server/start/start_master.go

```diff
@@ -754,7 +754,6 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 
 	// no special order
 	if configapi.IsBuildEnabled(&oc.Options) {
-		oc.RunBuildPodController()
```

Comment:
the loop above is for the new style of controllers (where they live in an array and initialized via a function in that array) (ala kube). We have not yet changed the config change controller to initialize in that way.